src/index.js

```diff
@@ -134,16 +133,38 @@ async function callOracle (
     signer
   )
 
+  // Wait until the network has heen established
+  await provider.ready
+
+  // Check if network supports EIP1559
+  const SUPPORTS_EIP1559 = Boolean(await provider.getBlock("latest").baseFee)
+
+  // Calculate fees
+  const feeData = await provider.getFeeData()
+
+  let OVERRIDES
+  if (SUPPORTS_EIP1559){
+    OVERRIDES = {
+      gasLimit: 1400000,
+      maxFeePerGas: feeData.maxFeePerGas,
+      maxPriorityFeePerGas: feeData.maxPriorityFeePerGas
+    }
+  } else {
+    OVERRIDES = {
+      gasPrice: feeData.gasPrice, gasLimit: 1400000
```

Comment:
just out for curiosity where are we taking this  gas limit from? or is just a value that we know  it works?

Reply:
I keep unchanged from the previous version that Oliver chose.